build.gradle

```diff
@@ -37,44 +35,30 @@ buildscript {
         maven {
             url = 'https://libraries.minecraft.net/'
         }
-        mavenCentral()
-        jcenter()
     }
-
     dependencies {
-        classpath group: 'com.github.ImpactDevelopment', name: 'ForgeGradle', version: '3.0.115'
```

Comment:
Is this added back somewhere else? If not, how does building for vanilla work?

Reply:
so basically, it builds vanilla by building forge but remapping to notch instead of searge. Also no ForgeGradle dep at all even for forge (that's what arch-loom is for)